common_features.mk

```diff
@@ -265,8 +265,15 @@ endif
 ifeq ($(strip $(SPLIT_KEYBOARD)), yes)
     OPT_DEFS += -DSPLIT_KEYBOARD
     QUANTUM_SRC += $(QUANTUM_DIR)/split_common/split_flags.c \
-                $(QUANTUM_DIR)/split_common/split_util.c
-    QUANTUM_LIB_SRC += $(QUANTUM_DIR)/split_common/i2c.c
-    QUANTUM_LIB_SRC += $(QUANTUM_DIR)/split_common/serial.c
+                   $(QUANTUM_DIR)/split_common/split_util.c
+    ifneq ($(strip $(CUSTOM_TRANSPORT)), yes)
```

Comment:
Honestly, I'm not really a fan of this.  

Mostly, because this basically re-introduces the issues that the split common code was trying to solve: fragmentation of the code base, and differing versions.